ogre2/include/gz/rendering/ogre2/Ogre2RenderEngine.hh

```diff
@@ -54,7 +54,7 @@ namespace ignition
     //
     // forward declaration
     class Ogre2RenderEnginePrivate;
-    class Ogre2IgnHlmsSphericalClipMinDistance;
+    class Ogre2GzHlmsSphericalClipMinDistance;
```

Comment:
Not this PR's fault, but I wonder how downstream applications are expected to use the `Ogre2RenderEngine.hh` header if the `Ogre2IgnHlmsSphericalClipMinDistance` header isn't installed.